addons/account/models/account_move.py

```diff
@@ -2222,6 +2222,9 @@ def name_get(self):
             result.append((move.id, name))
         return result
 
+    def _suggested_import_sheet_names(self):
```

Comment:
remove all the `_suggested_import_sheet_names` we'll set the sheet to import as the first tab. (and explanations in second tab)